\doxysection{Command\+Error Class Reference}
\hypertarget{classpip_1_1__internal_1_1exceptions_1_1_command_error}{}\label{classpip_1_1__internal_1_1exceptions_1_1_command_error}\index{CommandError@{CommandError}}


Inheritance diagram for Command\+Error\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=162pt]{df/d81/classpip_1_1__internal_1_1exceptions_1_1_command_error__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for Command\+Error\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=162pt]{d6/d09/classpip_1_1__internal_1_1exceptions_1_1_command_error__coll__graph}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when there is an error in command-line arguments\end{DoxyVerb}
 

Definition at line \mbox{\hyperlink{__internal_2exceptions_8py_source_l00286}{286}} of file \mbox{\hyperlink{__internal_2exceptions_8py_source}{exceptions.\+py}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
E\+:/pqc-\/engineering-\/ssec-\/23/venv/lib/python3.\+12/site-\/packages/pip/\+\_\+internal/\mbox{\hyperlink{__internal_2exceptions_8py}{exceptions.\+py}}\end{DoxyCompactItemize}
